datahub-web-react/src/app/entity/glossaryTerm/profile/GlossaryRelatedTerms.tsx

```diff
@@ -26,7 +23,8 @@ const Content = styled.div`
     flex-grow: 1;
 `;
 
-export default function GlossayRelatedTerms({ glossaryTerm }: Props) {
+export default function GlossayRelatedTerms() {
+    const { entityData }: any = useEntityData();
```

Comment:
i dont think this `: any` type annotation helps at all?

Reply:
agree!!, removed